OpenAI.Images.pas

```diff
@@ -437,4 +420,35 @@ destructor TAzureImageResponse.Destroy;
   inherited;
 end;
 
+{ TImagesAzureRoute }
+
+function TImagesAzureRoute.Create(ParamProc: TProc<TImageAzureCreateParams>): TAzureImageResponse;
+var
+  ARequest: TAzureImageRequest;
+begin
+  // First place the task with POST
+  ARequest := API.Post<TAzureImageRequest, TImageAzureCreateParams>('text-to-image', ParamProc);
+
+  // Repeat GET requesting the operations-endpoint until we have a "succeeded" response
+  while true do
```

Comment:
Why is there a cycle?

Reply:
For what? Can we remove the cycle and leave it to the discretion of the developer?

Reply:
With the first POST (text-to-image) using azure you only get a "id" as response. Then you have to query the operations-endpoint until you get the final image. 

So you would prefer to split it into two seperate calls from the developer instead of the loop?

Reply:
And yes, I saw it. It is not convenient to look at the entire code in the review on the git. Then, it is necessary to make check after the first request. That it was successfully completed and you can start the cycle to wait for readiness